test/e2e/testing-manifests/storage-csi/hostpath/README.md

```diff
@@ -1,4 +1,5 @@
 The files in this directory are exact copies of "kubernetes-latest" in
-https://github.com/kubernetes-csi/csi-driver-host-path/tree/v1.8.0/deploy/
+https://github.com/kubernetes-csi/csi-driver-host-path/tree/master/deploy/
```

Comment:
Why changing to "master"?  Code "master" is not stable.

Reply:
when we cut v1.12.1 release, the deploy still uses an ord image. please see https://github.com/kubernetes-csi/csi-driver-host-path/blob/v1.12.1/deploy/kubernetes-1.26/hostpath/csi-hostpath-plugin.yaml#L222. I'm not sure if the [PR](https://github.com/kubernetes-csi/csi-driver-host-path/pull/489) should be cherry-picked to release-1.12 branch. And I'm not sure if we should cut a new release for this because it's not a bug fix for the driver itself.

Reply:
We are behind updating the images.  I think we should use the latest stable branches.

Reply:
@xing-yang  I cherry-pick https://github.com/kubernetes-csi/csi-driver-host-path/pull/489 into release-1.12. please review https://github.com/kubernetes-csi/csi-driver-host-path/pull/497